spec/features/product_versioning_spec.rb

```diff
@@ -44,7 +44,7 @@
     visit "/cases/#{investigation.pretty_id}/products"
 
     expect(page).to have_selector("h3.govuk-heading-m", text: product.name)
-    expect(page).to have_selector("dd.govuk-summary-list__value", text: "psd-#{product.id}_#{investigation.date_closed.to_i} - The PSD reference for this version of the product record - as recorded when the case was closed: #{investigation.date_closed.to_s(:govuk)}.")
+    expect(page).to have_selector("dd.govuk-summary-list__value", text: "psd-#{product.id}_#{investigation.date_closed.to_i} - The PSD reference for this version of the product record - as recorded when the case was closed: #{investigation.date_closed.to_fs(:govuk)}.")
```

Comment:
We don't use `to_fs` anywhere else in the codebase, but we do use the longer form `to_formatted_s` quite a bit, so it might be preferable to keep the convention?

```suggestion
    expect(page).to have_selector("dd.govuk-summary-list__value", text: "psd-#{product.id}_#{investigation.date_closed.to_i} - The PSD reference for this version of the product record - as recorded when the case was closed: #{investigation.date_closed.to_formatted_s(:govuk)}.")
```

Reply:
👍🏻